src/commands/package.js

```diff
@@ -116,3 +112,102 @@ export default asyncCommand({
 	}
 });
 
+
+function getZimletLabelDescription() {
+	const cwd = process.cwd();
+	const intlDir = path.resolve(cwd, 'src', 'intl');
+
+	try {
+		const content = JSON.parse(
+			fs.readFileSync(path.resolve(intlDir, 'en_US.json'))
+		);
+
+		if (!(content.zimlet?.label && content.zimlet?.description)) {
+			throw new Error('label or description not found');
+		}
+		return {
+			label: content.zimlet.label,
+			description: content.zimlet.description
+		};
+	}
+	catch (err) {
+		warn(`Failed to read src/intl/en_US.json file: ${err}. \nMake sure below content present in src/intl/en_US.json file \n
+		"zimlet": {
+			"label": "<zimlet label>",
+			"description": "<zimlet description>"
+		} \nAs a fallback we are using strings from package.json which is deprecated.\n`);
+		return null;
+	}
+
+}
+
+function createLocalizationFiles(zimletName, label, description) {
+	const cwd = process.cwd();
+	const intlDir = path.resolve(cwd, 'src', 'intl');
+	let intlFiles;
+
+	// Add properties file for description and label
+	// As admin console requires properties file to display label and desciption
```

Comment:
this description should be moved to line no 143

Reply:
Done.